apd/add_json_attribute.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/ruby -w
+
+require 'json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.